aviary/wren/data.py

```diff
@@ -84,9 +84,12 @@ def __init__(
                 n_classes = np.max(self.df[target].values) + 1
                 self.n_targets.append(n_classes)
 
-    def __len__(self):
+    def __len__(self) -> int:
         return len(self.df)
 
+    def __repr__(self) -> str:
```

Comment:
Dataclasses now print like this:

```py
WyckoffData(df=(68, 6), task_dict={'E_f': 'regression'})
```

Reply:
I guess we could include the column names, not just `df.shape`?

Reply:
column names and then number of data points sounds more informative

Reply:
Repr with latest changes:

WyckoffData(cols=[material_id, composition, E_f, wyckoff, lattice, sites], len=68, task_dict={'E_f': 'regression'})

Reply:
I removed the quotes from col names to keep it short. Not sure if you approve?